dangerfile.js

```diff
@@ -1,15 +1,8 @@
-const { schedule, danger } = require('danger');
-const {
-    checkForRelease,
-    checkForNewDependencies,
-    checkForLockfileDiff,
-    checkForTypesInDeps
-} = require('danger-plugin-yarn');
+const { danger, warn } = require('danger');
+const { includes } = require('lodash');
 
-schedule(async () => {
-    const packageDiff = await danger.git.JSONDiffForFile('package.json');
-    checkForRelease(packageDiff);
-    checkForNewDependencies(packageDiff);
-    checkForLockfileDiff(packageDiff);
-    checkForTypesInDeps(packageDiff);
-});
\ No newline at end of file
+const hasPackageChanges = includes(danger.git.modified_files, 'package.json');
+const hasLockfileChanges = includes(danger.git.modified_files, 'yarn.lock');
+if (hasPackageChanges && !hasLockfileChanges) {
+    warn('There are package.json changes with no corresponding lockfile changes');
+}
```

Comment:
do we want to warn about this case as well?
```js
if (!hasPackageChanges && hasLockfileChanges)
```

Reply:
@ChaoticTempest No, because package.json can have changes that are not related to dependencies.